flake.nix

```diff
@@ -0,0 +1,103 @@
+{
+  description = "nixpkgs-graph-explorer flake";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    nixpkgs_old_cypress.url =
+      "github:NixOS/nixpkgs?rev=bf972dc380f36a3bf83db052380e55f0eaa7dcb6";
+    flake-utils.url = "github:numtide/flake-utils";
+    devshell = {
+      url = "github:numtide/devshell";
```

Comment:
I think we should use devenv instead. There are plans to deprecate `devshell`: https://github.com/numtide/devshell/issues/242#issuecomment-1464001725
Plus with `devenv` we should not need docker for development. 

I can help with it, probably it's better to create a new PR after this gets merged.

Reply:
https://github.com/tweag/nixpkgs-graph-explorer/issues/40